contracts/Treasury.sol

```diff
@@ -111,10 +105,10 @@ contract Treasury is Context, ReentrancyGuard {
      */
     function removeWhitelistedToken(address _token) external onlyGovernor {
         require(_whitelistedTokens.remove(_token), "remove-from-list-failed");
-        require(_cTokenList.remove(cTokens[_token]), "remove-from-list-failed");
-        IERC20(_token).safeApprove(cTokens[_token], 0);
-        delete cTokens[_token];
-        delete cTokens[_token];
+        require(_cometList.remove(comets[_token]), "remove-from-list-failed");
+        IERC20(_token).safeApprove(comets[_token], 0);
+        delete comets[_token];
+        delete comets[_token];
```

Comment:
why deleting twice?

Reply:
Good catch. One is suppose to delete from whitelist.